src/superu_data_collector/cli.py

```diff
@@ -0,0 +1,104 @@
+from argparse import ArgumentParser
+from logging import getLogger, Formatter, Logger, StreamHandler
+from logging import DEBUG, INFO
+from pathlib import Path
+from superu_data_collector.scraper_superu import SuperuScraper
+from superu_data_collector.connector_nodriver import ConnectorNodriver
+from typing import List, Dict
+import time
```

Comment:
Could you please reorganise the dependencies to group them by categories?

If you look at https://github.com/dataforgoodfr/offseason-proteines-resilientes/blob/689f06c3bc0bd4a03a8d05a583f512aba5d5e149/src/scraper_auchan/cli.py#L1-L7 for example, I grouped the standard libraries together, then the third-party dependencies, and finally, the local ones.